scripts/great-renaming.js

```diff
@@ -58,7 +58,7 @@ const rename = {
   users: 'user',
   utils: 'util',
   versions: 'version',
-  'video-fields': 'video-field',
+  'video-fields': 'oembed-field',
```

Comment:
I'm not sure if this is okay.

Reply:
Oh it's fine, this script could be removed at this point, it's historical.